docs/docs/recipes/integrate-logto/fragments/_configure-post-sign-out-redirect-uri.mdx

```diff
@@ -0,0 +1,9 @@
+<p>
+  Let&apos;s switch to the Application details page of Admin Console in this section. Add a Post
+  Sign-out Redirect URI <code>{props.postSignOutRedirectUri}</code> and click &quot;Save
+  Changes&quot;.
+</p>
+
+<img alt="Post Sign-out Redirect URI in Admin Console" src={props.figureSrc} width="600px" />
+
+[Post Sign-outRedirect URI](https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout) is an OAuth 2.0 concept which implies the location should redirect after signing out.
```

Comment:
Feels redundant here.  As we will have hints in the console. Could apply the ref link directly at the above sentence. 

Add a [Post Sign-outRedirect URI](https://openid.net/specs/openid-connect-rpinitiated-1_0.html#RPLogout)

Reply:
Per offline discussion, this part follows the _config-redirect-uri.mdx pattern.
Leave it now.